plugin/al/mayautils/AL/maya/utils/NodeHelper.h

```diff
@@ -735,6 +757,12 @@ class NodeHelper
   AL_MAYA_UTILS_PUBLIC
   static MObject addTimeAttr(const char* longName, const char* shortName, const MTime& defaultValue, uint32_t flags);
 
+  /// \brief  inherit in this node type a time attribute from a base node type.
+  /// \param  longName  long name for the attribute
+  /// \param  flags  a bitfield containing a mask of the AttributeFlags enumeration. Describes if the attribute is an input/output/etc
+  AL_MAYA_UTILS_PUBLIC
+  static void inheritTimeAttr(const char* longName, uint32_t flags);
```

Comment:
Aren't all of these gong to be the same code? Is there anything really specific about the fact it's a time v.s. Int32 attr?

Reply:
I may not be getting the question, but if you compare addTimeAttr vs inheritTimeAttr you will notice one will register attribute and create it on a node, the second one will inherit already existing attribute from the base class and register it internally to AL plugin.

Reply:
No, I mean that the code for inheritTimeAttr / inheritInt32Attr / inheritFloatAttr / etc, would all end up being identical. You could just do this with a single inheritAttr method (I think the inheritFileAttr is possibly the only exception)